Sources/Widgets/Widgets3D/ResliceCursorWidget/state.js

```diff
@@ -108,7 +108,8 @@ export default function generateState() {
     .addField({ name: 'activeRotationPointName', initialValue: null })
     .addField({ name: 'image', initialValue: null })
     .addField({ name: 'activeViewName', initialValue: '' })
-    .addField({ name: 'sphereRadius', initialValue: 5 })
+    .addField({ name: 'sphereRadius', initialValue: 20 })
```

Comment:
it seems quite big, for a default, no ?

Reply:
Indeed... That was a test. It should have not been pushed.